spec/Section 5 -- Validation.md

```diff
@@ -1134,6 +1164,43 @@ fragment catInDogFragmentInvalid on Dog {
 ```
 
 
+##### Tagged Spreads In Tagged Scope
+
+In the scope of a tagged type, the only valid tagged type
+fragment spread is one that applies to the same type that
+is in scope.
+
+For example
+
+```graphql example
+fragment beingFragment on Being {
+  ... on Being {
+    dog {
+      barkVolume
+    }
+  }
+}
+```
+
+and the following is invalid
+
+```graphql counter-example
+fragment beingFragment on Being {
+  ... on Terran {
+    dog {
+      barkVolume
+    }
+  }
+}
+```
+
+This counter-example may be surprising since Being is covariant to Terran
```

Comment:
Interesting that this note actually applies to object types that appear to be covariant as well. I think it's fine to leave it just here.

My thought was perhaps we want to merge this with the rule above as "Concrete Spreads in Concrete Scope" but perhaps that's more generalized but also harder to understand. The separate cases here seem just fine